.evergreen/run-enterprise-auth-tests.sh

```diff
@@ -23,5 +23,10 @@ echo "Install dependencies"
 export BUNDLE_GEMFILE=gemfiles/mongo_kerberos.gemfile
 bundle install --gemfile="$BUNDLE_GEMFILE"
 
+unset CI
 echo "Running tests"
 bundle exec rspec spec/enterprise_auth -fd
+bundle exec rspec spec/spec_tests/uri_options_spec.rb -fd
+bundle exec rspec spec/spec_tests/connection_string_spec.rb -fd
+bundle exec rspec spec/mongo/uri/srv_protocol_spec.rb -fd
+bundle exec rspec spec/mongo/uri_spec.rb -fd
```

Comment:
I opted to only run files that have relevant tests to keep CI as speedy as possible. That means that anybody who adds kerberos-dependent tests to a different file will have to remember to include that file.